bindings/nodejs/lib/client/client.ts

```diff
@@ -68,7 +69,10 @@ import {
     CongestionResponse,
     UtxoChangesResponse,
     UtxoChangesFullResponse,
+    ICommitteeResponse,
```

Comment:
While we're touching these files can we remove the `I` prefixes

Reply:
IIRC correctly @Thoralf-M thinks they should be added.

Reply:
Well only to have it consistent, because we added it for interfaces in the past, personally I'm also fine with removing it everywhere :joy:

Reply:
removed the `I` prefixes